### ISSUE TITLE
ForeCatch_2018 (for example) has catch large than the specified value

### ISSUE THREAD

**Issue by andrew-edwards** [cgrandin/hake-assessment#368]
After source("all.r"), do:
```
temp = SSgetMCMC("../../models/2018.40_base_model/mcmc/forecasts/forecast-year-2018/05-597500", writecsv=FALSE)$model1
temp$ForeCatch_2018[1786:1790]
```

gives
```
597500 597500 597501 597500 597500
```

So why would there be a 597501 value, when the catch has been specified to be a constant 597500? Other catch policies get similar (worse) issues.

This is why the numbers come out wrong in Table i:

![image](https://user-images.githubusercontent.com/6977190/35935773-44d9fc30-0bf6-11e8-93b2-fe3c14bbd7d0.png)

It ends up in the table via the line
```
    out[1] <- max(x[, paste0("ForeCatch_", yr)])
```
in calc.risk().

Simple solution:
1. we can change that line to use the specified catch.level. 

But it'd be good to confirm that SS3 is doing what we think.

**Comment by andrew-edwards** [comment]
![image](https://user-images.githubusercontent.com/6977190/35936115-33643500-0bf7-11e8-8290-6178bcdff5f0.png)

turns into

![image](https://user-images.githubusercontent.com/6977190/35936138-42f02240-0bf7-11e8-93d6-bfd8fc54da7b.png)

So much more than a rounding error.

**Comment by andrew-edwards** [comment]
Should the last three numbers of the (Prob 2019 relative fishing intensity >100%) column all be 50%?

They were last year in the equivalent table.

**Comment by andrew-edwards** [comment]
The row h catch in 2019 values are:
```
temp = SSgetMCMC("../../models/2018.40_base_model/mcmc/forecasts/forecast-year-2019/08-stable-catch", writecsv=FALSE)$model1

> head(sort(temp$ForeCatch_2019), n=20)
 [1] -1.96427e+32 -1.10709e+32 -6.42555e+31  1.89554e+05  2.86219e+05
 [6]  5.19911e+05  5.40859e+05  5.99263e+05  6.17511e+05  6.26008e+05
[11]  6.26918e+05  6.26954e+05  6.26954e+05  6.26954e+05  6.26954e+05
[16]  6.26954e+05  6.26954e+05  6.26954e+05  6.26954e+05  6.26954e+05
> 

> tail(sort(temp$ForeCatch_2019), n=30)
 [1] 626954 626954 626954 626954 626954 626954 626954 626954 626954 626954
[11] 626954 626955 626955 626956 626957 626960 626973 626983 626986 627008
[21] 627047 627077 627175 627966 629403 629429 634265 636270 649560 715755
```

so quite a few of them are not the prescribed 626,954.

**Comment by andrew-edwards** [comment]
Chris to fix the table (using the input catch.levels). The relatively few outliers are not going to influence the medians or probabilities. 

Regarding my comment below the table (above), only row f has to have 50% in the penultimate column. The three 2017 assessment ones were 50% by coincedence (and these were 55% in 2016). And in the larger tables we correctly get 100% values where we should.

**Comment by cgrandin** [comment]
Fixed in commit c192d0f2e081a29c6f35f5bf20952ddde24ef80f

**Comment by cgrandin** [comment]
You need to delete the Rdata file from the base model and run build() again!!!
